### PR TITLE
Changed style of Doctor Class Feats

### PR DESCRIPTION
**Pull request by bleehu**
and added flavor text to the Doctor class

	modified:   02_Classes.txt

### CODE REVIEW

**Review by Turtlelord26 — COMMENTED** [review]
What do you think about copying the prerequisite format from the rest of the feats?

Aside from that, my only comments are
* This looks good
* 'Errant twitch' in lines 149 and 190 is repetitive.

**Comment by bleehu** [comment]
My concern with copying the prereq format is that we have a lot of feats with only one prereq, and the parser needs at least 2 to realize that we're making a list.

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -143,8 +143,11 @@ Skill Proficiencies:
 
 
 =====   DOCTOR   =====
-You know how or why something works [most likely humans / humanoid bodies), 
-and put that knowledge to good use by supporting your team.
+
+    You know the secrets of life and death. The very continued existance of
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: sp:"existence"

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -143,8 +143,11 @@ Skill Proficiencies:
 
 
 =====   DOCTOR   =====
-You know how or why something works [most likely humans / humanoid bodies), 
-and put that knowledge to good use by supporting your team.
+
+    You know the secrets of life and death. The very continued existance of 
+those around you depend on you striking the right balance between heal and harm.
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: gr:"depends"
Also, I'd go with "on whether you strike the right balance between heal and harm."

File: CharacterCreation/02_Classes.txt [hunk 3]
```diff
@@ -143,8 +143,11 @@ Skill Proficiencies:
 
 
 =====   DOCTOR   =====
-You know how or why something works [most likely humans / humanoid bodies), 
-and put that knowledge to good use by supporting your team.
+
+    You know the secrets of life and death. The very continued existance of 
+those around you depend on you striking the right balance between heal and harm. 
+One errant twitch of your hands could mean the difference between saving the day
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: "one errant twitch" is a vivid enough phrase that I'd only use it once in a short passage like this. m/b choose whether to use it here or in the Adrenaline Junkie flavor text.

**Review by rosebrallier — APPROVED** [review]
Shipit! The new prose really captures the central conflict of the class (to f- people up or un-f mah frands)

**Comment by rosebrallier** [comment]
Oh! Just to make sure, the 2nd person is an intentional deviation from the norm, yes? (It works really well imo)

**Pull request merged**